tools/cythonize.py

```diff
@@ -100,6 +100,24 @@ def process_tempita_pyx(fromfile, tofile):
         f.write(pyxcontent)
     process_pyx(pyxfile, tofile)
 
+
+def process_tempita_pxi(fromfile, tofile):
+    try:
+        try:
+            from Cython import Tempita as tempita
```

Comment:
1) If you could put a link to the comment in your comment, that would be great.

2) I read that Google discussion.  Where does it mention this unreliability?  If anything, it seems to suggest that this usage is widerspread.